tools/gromacs/sim.xml

```diff
@@ -119,67 +122,120 @@
     <inputs>
         <param argument="gro_input" type="data" format='gro' label="GRO structure file"/>
         <param argument="top_input" type="data" format='top' label="Topology (TOP) file"/>
-        <conditional name="cpt_inp">
-            <param name="cpt_bool" type="select" label="Use a checkpoint (CPT) file" help="CPT file from a previous MD run">
-                <option value="yes">Continue simulation from a CPT file</option>
-                <option value="no" selected="true">No CPT input</option>
+
+        <section name="inps" title="Inputs" expanded="false">
+        
+            <!-- CPT inp -->
+            <conditional name="cpt_inp">
+                <param name="cpt_bool" type="select" label="Use a checkpoint (CPT) file" help="CPT file from a previous MD run">
+                    <option value="yes">Continue simulation from a CPT file</option>
+                    <option value="no" selected="true">No CPT input</option>
+                </param>
+                <when value="yes">
+                    <param argument="cpt_in" type="data" format='cpt' label="Checkpoint file"/>
```

Comment:
@simonbray why can this be not simply optional="true" and remove the entire conditional? Conditional with just one single param inside are not useful imho.

Reply:
the results of the boolean are also fed into the config file. I don't know if there is another way of doing this.

Reply:
@simonbray can you not do something like that for the boolean?

```
#if $file:
   set boolean
#end if
```
In the config file. So only set the boolean when the file was selected?

Reply:
@bgruening, done